system-contracts/bootloader/bootloader.yul

```diff
@@ -1662,12 +1661,11 @@ object "Bootloader" {
                     32
                 )
 
-                // If the transaction failed, the most likely reason is that there
-                // was not enough gas. That's why we do the nearCallPanic to stop the near call frame.
+                // If the transaction failed, either there was not enough gas or compression is malformed.
                 if iszero(success) {
                     debugLog("compressor call failed", 0)
                     debugReturndata()
-                    nearCallPanic()
+                    assertionError("Compressor failed")
```

Comment:
this is a breaking change to the server: we currently expect that if the user does not have enough gas, the transaction will be reverted and not rejected.

Reply:
@StanislavBreadless lets' acknowledge this issue then :(
I don't want to make solution in which I am not fully sure to not make the bug in the contest